docker-compose.yml

```diff
@@ -37,6 +43,26 @@ services:
       - network.host=0.0.0.0
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - ELASTICSEARCH_PROTOCOL=https
+      - ELASTICSEARCH_IP=elasticsearch
+      - ELASTICSEARCH_PORT=9200
+      - SECURITY_ENABLED=yes
+      - SECURITY_ADMIN_USER=wazuh_admin
+      - SECURITY_ADMIN_PASS=admin_pass
+      - SECURITY_ELASTIC_PASSWORD=elastic_pass
+      - SECURITY_KIBANA_USER=service_kibana
+      - SECURITY_KIBANA_PASS=kibana_pass
+      - SECURITY_LOGSTASH_USER=service_logstash 
+      - SECURITY_LOGSTASH_PASS=logstash_pass
+      - SECURITY_CA_PASSPHRASE=ca_pass
+      - SECURITY_CERTIFICATE_DNS=elasticsearch
+      - SECURITY_CA_PEM=server.TEST-CA-signed.pem
+      - SECURITY_CA_KEY=server.TEST-CA.key
+      - SECURITY_CA_TRUST=server.TEST-CA-signed.pem
+      - SECURITY_EXPECTED_HOSTNAME=elasticsearch
```

Comment:
`SECURITY_EXPECTED_HOSTNAME` should be better as `SECURITY_MAIN_NODE`. I understand that this node is the one that performs the `curls` when we are in a cluster mode.

Reply:
You are rigth, changed.